as/cNFT/assembly/core.ts

```diff
@@ -64,7 +94,21 @@ export function nft_transfer(token_id: TokenId, receiver_id: AccountId): void {
 
 }
 
-
+/**
+ * Burn a token
+ *
+ *  **Note:** In order to burn a token, user must be the creator and the owner of the token
+ *
+ * **Basic usage example:**
+ *
+ * Assume we need to burn a token with the token id = `jenny911038`,
+ *
+ * ```
+ * burn_design("jenny911038");
+ * ```
+ *
+ * @param token_id ID of the token that needs to burn
+ */
 @nearBindgen
 export function burn_design(token_id: TokenId): void {
```

Comment:
Maybe add an explanation to what burn_design does exactly, because it's not deleting the token completely